packages/api/src/middlewares/headersMiddleware.ts

```diff
@@ -0,0 +1,16 @@
+import { NextFunction, Request, Response } from "express";
+
+const headers = {
+    "Content-Security-Policy": "default-src 'none'",
+    "Strict-Transport-Security": "max-age=63072000; includeSubDomains; preload",
+    "X-Frame-Options": "DENY",
+    "X-Content-Type-Options": "nosniff",
+    "Cache-Control": "max-age 1800",
+};
+
+export const headersMiddleware = (req: Request, res: Response, next: NextFunction) => {
+    for (const [headerKey, headerValue] of Object.entries(headers)) {
```

Comment:
Tu ne veux pas l'utiliser mon .forEach :joy:

Reply:
Non mais le lien que tu m'as passé il va dans mon sens donc je n'ai plus aucun scrupule

Reply:
quelle lien ?

Reply:
https://leanylabs.com/blog/js-forEach-map-reduce-vs-for-for_of/

Reply:
oui exactement celui-là. C'est en ops/s donc on veut en avoir le + possible